R/cluster.R

```diff
@@ -0,0 +1,30 @@
+#' cluster based on vectors
+#' @export
+cluster <- function(..., type = "kmeans", n_cluster = 3) {
+  data <- list(...) %>% as.data.frame() %>% as.matrix()
+  na_removed <- data %>% na.omit()
+  if(nrow(na_removed) == 0) {
+    stop("no valid data to cluster")
```

Comment:
This message is a little too terse to understand what it is quickly. We could be a little more explanatory like this.
"not enough valid data to perform clustering"